XYZDrivers/src/main/webapp/error.jsp

```diff
@@ -1,17 +1,18 @@
-<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
-<%@page contentType="text/html" pageEncoding="UTF-8"%>
 <!DOCTYPE html>
+<!--
+To change this license header, choose License Headers in Project Properties.
+To change this template file, choose Tools | Templates
+and open the template in the editor.
+-->
 <html>
     <head>
         <title>Fatal Error</title>
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <link rel="stylesheet" type="text/css" href="node_modules/bootstrap/dist/css/bootstrap.min.css" />
-        <link rel="stylesheet" type="text/css" href="styles/main.css" />
     </head>
     <body>
-        <jsp:include page="nav.jsp"></jsp:include>
```

Comment:
Why remove the nav? Would make it easier for the user to get back to where they were?

Reply:
Good catch, accidentally deleted!